forem.gemspec

```diff
@@ -31,4 +31,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'select2-rails', '~> 3.5.4'
   s.add_dependency 'friendly_id', '~> 5.0.0'
   s.add_dependency 'cancancan', '~> 1.7'
+  s.add_dependency 'jbuilder'
```

Comment:
I would personally lock the jbuilder version here, just in case of a major version bump that changes things in a breaking way.